### PR TITLE
Develop a To do app!

### PR DESCRIPTION
**Pull request by ngeelani48**
**In This Branch:**

1. Create a `Components` folder for different functionalities.
2. Add Functionality for adding a task.
3. Add functionality for deleting or editing a task
4. Update the `README.md` file.
5. Add Styling to the ToDo list app.
6. Fix some linters issues.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+- GitHub: [@ngeelani48](https://github.com/ngeelani48)
+- Twitter: [@twitterhandle](https://twitter.com/ngeelani48)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/m-n-geelani-3446097275/)
+
+👤 **Sane Myburg**
+
+- GitHub: [@SaneMyburg](https://github.com/SaneMyburg)
+- Twitter: [@SAneMyburg](https://twitter.com/SaneMyburg)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/SaneMyburg/)
+
```
Comment (Tafloninno) [inline README.md]: - [ ]  Kindly update your README file

File: src/components/TodoItem.jsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
+/* eslint-disable */
```
Comment (Tafloninno) [inline src/components/TodoItem.jsx:1]: - [ ] I suggest enabling linters by removing  this comment

**Review by Tafloninno — APPROVED** [review]
[# STATUS:
APPROVED green_circle: :clap:
Hello @TEAM
Your project is complete! There is nothing else to say other than... it's time to merge it shipit:
Congratulations! Keep up the good
work
HIGHLIGHTS:
-   [x] Your repo is professional
-   [x] Descriptive P.R °
-   [x] linters are passing
-   [x] Professional README
100
-   [x] Your work is well documented

File: README.md (hunk 1 above)
Comment (ngeelani48) [reply README.md]: @Tafloninno Thanks for having a deep look, I will make these changes.

File: src/components/TodoItem.jsx (hunk 2 above)
Comment (ngeelani48) [reply src/components/TodoItem.jsx:1]: Thanks, And I will do it.

**Pull request merged**